src/main/java/com/SAS/team/Team.java

```diff
@@ -168,8 +170,22 @@ public String getName() {
      *
      * @param name
      */
-    public void setName(String name) {
+    public boolean setName(String name) {
+        if (name == null || name.length() == 0) {
```

Comment:
Maybe consider this test:
if (name != null && !name.trim().isEmpty()) {

so that spaces wouldn't be alowed.